pinot-core/src/main/java/com/linkedin/pinot/core/minion/rollup/aggregate/SumValueAggregator.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (C) 2014-2018 LinkedIn Corp. (pinot-core@linkedin.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.pinot.core.minion.rollup.aggregate;
+
+import com.linkedin.pinot.common.data.MetricFieldSpec;
+
+
+/**
+ * Sum value aggregator
+ */
+public class SumValueAggregator implements ValueAggregator {
+
+  @Override
+  public Object aggregate(Object value1, Object value2, MetricFieldSpec metricFieldSpec) {
+    Object result;
+    switch (metricFieldSpec.getDataType()) {
+      case LONG:
+        result = ((Number) value1).longValue() + ((Number) value2).longValue();
+        break;
+      case FLOAT:
+        result = ((Number) value1).floatValue() + ((Number) value2).floatValue();
+        break;
+      case DOUBLE:
+        result = ((Number) value1).doubleValue() + ((Number) value2).doubleValue();
+        break;
+      default:
+        throw new IllegalArgumentException("Unsupported metric type for SUM aggregator" + metricFieldSpec.getDataType());
```

Comment:
Add a space right after the string "aggregator"

Reply:
good catch